src/json.ts

```diff
@@ -123,7 +83,28 @@ export const JsonStruct = define<Json>('Json', (value, context) => {
  * @returns Whether the value is a valid {@link Json} value.
  */
 export function isValidJson(value: unknown): value is Json {
-  return is(value, JsonStruct);
+  try {
+    getValidatedAndSanitizedJson(value);
+    return true;
+  } catch {
+    return false;
+  }
+}
+
+/**
+ * Validate and return sanitized JSON.
+ *
+ * Note:
+ * This function use sanitized JsonStruct for validation
+ * that applies stringify and then parse of a value provided
+ * to ensure that there are no getters which can have side effects
+ * that can cause security issues.
+ *
+ * @param value - JSON structure to be processed.
+ * @returns Sanitized JSON structure.
+ */
+export function getValidatedAndSanitizedJson<Type>(value: unknown): Type {
```

Comment:
```suggestion
export function getValidatedJson<Type>(value: unknown): Type {
```
Or
```suggestion
export function getSafeJson<Type>(value: unknown): Type {
```

Reply:
Given that we already used term `valid/validated` and it was not secure, while now we have two steps: validation + sanitization, I would like to avoid using `getValidatedJson`.

To me, your other suggestion `getSafeJson` sounds good to me, so I updated the function name with it.